client/lib/create-selector/index.js

```diff
@@ -7,9 +7,27 @@ import shallowEqual from 'react-pure-render/shallowEqual';
 /**
  * Constants
  */
+
+/**
+ * Defines acceptable argument types for a memoized selector when using the
+ * default cache key generating function.
+ *
+ * @type {Array}
+ */
 const VALID_ARG_TYPES = [ 'number', 'boolean', 'string' ];
 
 /**
+ * Default behavior for determining whether current state differs from previous
+ * state, which is the basis upon which memoize cache is cleared. Should return
+ * a value or array of values to be shallowly compared for strict equality.
+ *
+ * @type   {Function}
+ * @param  {Object}    state Current state object
+ * @return {(Array|*)}       Value(s) to be shallow compared
+ */
+const DEFAULT_GET_DEPENDENTS = ( state ) => state;
```

Comment:
Might be worthwhile to add in a warning in dev mode if we forget to pass in dependents.